assets/pages/new-contributions-landing/components/SetPassword/setPassword.scss

```diff
@@ -0,0 +1,48 @@
+.set-password {
+  .form__input--email {
+    pointer-events:none;
+    background-color: #dcdcdc;
+
+  }
+  
+  .form__field--contribution-password {
+    margin-top: $gu-v-spacing;
+  }
+
+  .form__submit-button--create-account {
+    width: 100%;
+    position: relative;
+  }
+
+  .svg-arrow-right-straight {
```

Comment:
why do we need this? can we not re-use styles from the `.form__submit-button` on the new landing page? (there the mechanism for positioning the SVG is just padding on the button, and making the svg `display: inline-block`)

Reply:
@joelochlann I've now refactored this so that both buttons are instances of `<ButtonWithRightArrow />`

Reply:
so can we delete some of these styles?

Reply:
I've deleted the ones that weren't needed. The only ones left are the ones that override the base classes

Reply:
ah apart from this one. I'll delete it now

Reply:
If I delete this, it ends up looking like this:

![image](https://user-images.githubusercontent.com/2844554/46424670-af1c6a00-c731-11e8-8c91-10d75c3b448c.png)

basically the placement of the copy and arrow for the buttons here and the button on the landing page is a bit different, as one is a full width button with left aligned copy and the other is a "fit to size" button", and therefore won't have the same solution

Reply:
We could do this:

![image](https://user-images.githubusercontent.com/2844554/46424776-f145ab80-c731-11e8-88ef-c258c84f243d.png)

But that looks a bit odd, I imagine that they were designed full width for a reason

Reply:
Ah, good point!

So if you're using the button from the landing page, it already has `display: inline-flex` (as well as `align-items: center`, which in fact means that [this style](https://github.com/guardian/support-frontend/blob/91e84081a90942c6df3212ec82dc35054b02539e/assets/pages/new-contributions-landing/contributionsLanding.scss#L839) is redundant). This means all you should need is a `justify-content: space-between` to tell the children to spread themselves out to the edges if the button is wider than its content (i.e. there is space to spare).

At that point, you should just need the `width: 100%` and you have buttons that will work at the size or their contents, or at full width, without messing with the positioning of what's inside them